lightning-persister/src/lib.rs

```diff
@@ -87,34 +55,8 @@ impl FilesystemPersister {
 		self.path_to_channel_data.clone()
 	}
 
-	pub(crate) fn path_to_monitor_data(&self) -> PathBuf {
-		let mut path = PathBuf::from(self.path_to_channel_data.clone());
-		path.push("monitors");
-		path
-	}
-
-	/// Writes the provided `ChannelManager` to the path provided at `FilesystemPersister`
-	/// initialization, within a file called "manager".
-	pub fn persist_manager<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>(
-		data_dir: String,
-		manager: &ChannelManager<Signer, M, T, K, F, L>
-	) -> Result<(), std::io::Error>
-	where
-		M::Target: chain::Watch<Signer>,
-		T::Target: BroadcasterInterface,
-		K::Target: KeysInterface<Signer=Signer>,
-		F::Target: FeeEstimator,
-		L::Target: Logger,
-	{
-		let path = PathBuf::from(data_dir);
-		util::write_to_file(path, "manager".to_string(), manager)
-	}
-
-	/// Write the provided `NetworkGraph` to the path provided at `FilesystemPersister`
-	/// initialization, within a file called "network_graph"
-	pub fn persist_network_graph(data_dir: String, network_graph: &NetworkGraph) -> Result<(), std::io::Error> {
-		let path = PathBuf::from(data_dir);
-		util::write_to_file(path, "network_graph".to_string(), network_graph)
+	pub(crate) fn path_to_monitor_data(&self) -> String {
+			format!("{}{}monitors", self.path_to_channel_data, MAIN_SEPARATOR)
```

Comment:
Given this is used only once, could we instead inline it at the call site where it already uses `PathBuf`. That would prevent double separators, IIUC.  Not sure how we care about that, but no need for the method at very least.

Reply:
yeah that makes sense to me.  though in #1427 we will be using string keys to list monitors dir and read monitors so we might run into the string => PathBuf => string conversion fun again.